src/lasp_core.erl

```diff
@@ -554,10 +560,8 @@ bind_to(AccId, Id, Store, BindFun) ->
     bind_to(AccId, Id, Store, BindFun, ?READ).
 
 bind_to(AccId, Id, Store, BindFun, ReadFun) ->
-    Fun = fun({_, _, _, V}) ->
-        {ok, _} = BindFun(AccId, V, Store)
-    end,
-    lasp_process:start_link([[{Id, ReadFun}], Fun]).
+    TransFun = fun({_, _, _, V}) -> V end,
```

Comment:
I assumed it was safe to do so since the previous implementation only cared about the input value. 

Seems like `bind_to(S2, S1)` is the same as `map(S1, fun(X) -> X end, S2)`, so the transforming function just returns the input value, and the write function binds it to S2.